src/slack.ts

```diff
@@ -604,79 +628,174 @@ export class SlackBot {
         this.config = config.ChessterConfigDecoder.decodeJSON(
             JSON.stringify(require(this.configFile))
         )
-        this.token = this.config.slackTokens[this.slackName]
+
+        // Confusing naming; chesster is the name of adminSlack; lichess4545 is the name of actual chesster
+        // @ts-expect-error it thinks token can be undefined here but it's incorrect
+        this.token =
+            this.slackName === 'lichess4545'
```

Comment:
This stuff should be moved to main and pass along to this class, rather than parsed here.